src/main/java/Model/Utilities/Check.java

```diff
@@ -0,0 +1,55 @@
+package Model.Utilities;
+
+import Model.Game;
+import Model.Position;
+import Model.Pieces.Piece;
+import Model.Pieces.PieceType;
+
+import java.util.ArrayList;
+
+import static java.util.Objects.isNull;
+
+public class Check {
```

Comment:
why are we using a static class like this?

Reply:
I don't know what you mean